odoo/http.py

```diff
@@ -985,7 +985,33 @@ def generate_key(self, salt=None):
     def is_valid_key(self, key):
         return _base64_urlsafe_re.match(key) is not None
 
+    def get_nonexistent_session_identifiers(self, identifiers):
```

Comment:
```suggestion
    def get_missing_session_identifiers(self, identifiers):
```
Feel free to ignore. This is I believe more conventional. For instance `get_missing_ids` in `api.py`